external/kafka-0-10/src/main/scala/org/apache/spark/streaming/kafka010/ConsumerStrategy.scala

```diff
@@ -68,8 +67,7 @@ trait ConsumerStrategy[K, V] {
  * TopicPartition, the committed offset (if applicable) or kafka param
  * auto.offset.reset will be used.
  */
-@Experimental
```

Comment:
You just told me in the last PR to remove Experimental annotations from private classes.  This is private.  What's the actual rule?